packages/now-cli/src/util/deploy/get-deployment-by-id-or-host.ts

```diff
@@ -8,13 +8,13 @@ import {
 } from '../errors-ts';
 import mapCertError from '../certs/map-cert-error';
 
-type APIVersion = 'v5' | 'v10';
+type APIVersion = 'v12';
 
 export default async function getDeploymentByIdOrHost(
   client: Client,
   contextName: string,
   idOrHost: string,
-  apiVersion: APIVersion = 'v5'
+  apiVersion: APIVersion = 'v12'
```

Comment:
I'd suggest just using `string` type here.

```suggestion
  apiVersion = 'v12'
```